### ISSUE TITLE
Instantly crashing when opening

### ISSUE THREAD

**Issue by Hypertive** [Chatterino/chatterino2#2746]
My chatterino instantly crashes after opening

Whenver I launch it and xQc is live, the chat works for like 1-2 seconds then it doesn't respond. Note this seems to only happen when xQc is live because I have him as the only chat on chatterino.

    Examples:
    - I cannot chat
    - I have trouble viewing chat

  

    You can add screenshots to illustrate what you mean.
    Remove this section if you do not have any.

http://prntscr.com/12j7io3 (note this is because I changed my password recently, otherwise it would show chat messages for the first 1-2 seconds I open it then it crashes)

**OS and Chatterino Version**
  I don't remember if this happened before, but I know it happened a lot more in chatterino 2.3.0

    current version: 2.3.0
    OS info:       Windows Feature Experience Pack 120.2212.551.0
 OS BUILD: 19042.928
Version: 20H2
<!--

**Comment by apa420** [comment]
Duplicate of #2704 
tldr: update to latest nightly instead
